GOVERNANCE.md

```diff
@@ -64,6 +64,30 @@
   being actively covered by someone.
 * The on-call rotation is tracked at PagerDuty. The calendar is visible [here](https://pagerduty.github.io/addons/PDcal/index.html?iCalURL=https://cncf.pagerduty.com/private/e44caf2604ce6c5ccc616b7b84f99b94dc801dba4cceb8d71fb128338f75b9af/feed/PXU9KPH) or you can subscribe to the iCal feed [here](https://cncf.pagerduty.com/private/e44caf2604ce6c5ccc616b7b84f99b94dc801dba4cceb8d71fb128338f75b9af/feed/PXU9KPH).
 
+## Cutting a release
+
+* We do releases approximately every 3 months as described [here](CONTRIBUTING.md#release-cadence).
+* Decide on the somewhat arbitrary time that a release will occur.
+* Begin marshalling the ongoing PR flow in both this repo and
+  [data-plane-api](https://github.com/envoyproxy/data-plane-api). Try to push out any particularly
```

Comment:
Try to push out any particularly risky PRs
-> ask maintainers to hold off merging any particularly risky PRs
?